imod/mf6/drn.py

```diff
@@ -163,6 +165,21 @@ def _validate(self, schemata, **kwargs):
 
         return errors
 
+    def cleanup(
+        self, dis: Union[StructuredDiscretization, VerticesDiscretization]
+    ) -> None:
+        """
+        Clean up package inplace. This method calls
+        :func:`imod.prepare.cleanup.cleanup_drn`, see documentation of that
+        function for details on cleanup.
+
+        dis: imod.mf6.StructuredDiscretization | imod.mf6.VerticesDiscretization
+            Model discretization package.
+        """
+        dis_dict = {var: dis.dataset[var] for var in ["idomain"]}
```

Comment:
There is only 1 item in the list. You can skip for loop and directly declare the dictionary
`dis_dict = {var: dis.dataset["idomain"]}`

Reply:
Removed the dict comprehension